examples/shrink_grow_layout.rs

```diff
@@ -0,0 +1,108 @@
+use bevy::{
```

Comment:
Can you add an inner doc comment at the top here briefly describing what this example is for?